regolith/helpers/a_expensehelper.py

```diff
@@ -14,42 +14,45 @@
 )
 
 TARGET_COLL = "projecta"
-ALLOWED_TYPES = ["nsf", "doe", "other"]
-ALLOWED_STATI = ["proposed", "started", "finished", "back_burner", "paused",
-                 "cancelled"]
-MILESTONES_ALLOWED_STATI = ["proposed", "scheduled", "finished", "cancelled"]
+ALLOWED_TYPES = ["business", "travel"] # need to check all expense types.
+ALLOWED_STATI = ["submitted", "unsubmitted"]
 
 
 def subparser(subpi):
-    subpi.add_argument("name", help="A short but unique name for the projectum",
-                       default=None)
-    subpi.add_argument("lead", help="id of the group lead or tbd",
-                       default=None)
-    # Do not delete --database arg
-    subpi.add_argument("--database",
-                       help="The database that will be updated.  Defaults to "
-                            "first database in the regolithrc.json file."
+    subpi.add_argument("amount", help="expense amount",
                        )
-    # Do not delete --date arg
-    subpi.add_argument("--date",
-                       help="The begin_date for the projectum  Defaults to "
-                            "today's date."
+    subpi.add_argument("name", help="A short name for the expense",
+                       default=None
                        )
-    subpi.add_argument("-d", "--description",
-                       help="Slightly longer description of the projectum"
+    subpi.add_argument("-b", "--business",
+                       help="expense type is business"
```

Comment:
let user know that if -b not specified it will assume it is a travel expense.

Also, you will have to add code that sets business to True if -b is set.